MapboxNavigation/NavigationViewController.swift

```diff
@@ -687,6 +688,17 @@ extension NavigationViewController: NavigationServiceDelegate {
         return navigationComponents.allSatisfy { $0.navigationServiceShouldDisableBatteryMonitoring(service) }
     }
     
+    public func locationManagerDidChangeAuthorization(_ manager: CLLocationManager) {
+        let title = NSLocalizedString("ENABLE_PRECISE_LOCATION", bundle: .mapboxNavigation, value: "Enable precise location to  navigate", comment: "Label indicating precise location is off and needs to be turned on to navigate")
```

Comment:
Nit: there’s an extra space between “to” and “navigate”. Apparently UIKit doesn’t collapse extra spaces, so a gap shows up on screen.